### ISSUE TITLE
fix: assembly not found error in 3.0.0-RC.1 nuget package

### ISSUE THREAD

**Issue by ashikns** [microsoft/fluentui-blazor#573]
# 🐛 Bug Report

Installing the 3.0 RC1 package shows an error on the homepage while running:
![image](https://github.com/microsoft/fluentui-blazor/assets/6162428/7b2497b6-30ab-49ef-9c3a-a7f1f4f7bcaf)

preview.5 package works fine.

## 💻 Repro or Code Sample

Update nuget in project generated by template, and follow upgrade instructions.

## 🤔 Expected Behavior

Should refer to correct dll and render page.

## 😯 Current Behavior

Does not render.

## 🔦 Context

Trying out FluentUI for Blazor for internal application.

## 🌍 Your Environment

* OS & Device: Win 11 on Desktop (Ryzen 5800X, RTX 3080)
* Browser: Google Chrome, also tried Edge
* .NET and FAST Version: .Net 7.0, haven't touched FAST version so dunno

**Comment by vnbaaij** [comment]
This is caused by an error in the Emoji and package. For now, removing that package from your solution should make it work again

**Comment by LuohuaRain** [comment]
Should I remove `Microsoft.Fast.Components.FluentUI.Icons`?

**Comment by vnbaaij** [comment]
@LuohuaRain are you seeing this error?

**Comment by LuohuaRain** [comment]
@vnbaaij Yes. It got error with 

```csproj
<PackageReference Include="Microsoft.Fast.Components.FluentUI" Version="3.0.0-RC.1" />
<PackageReference Include="Microsoft.Fast.Components.FluentUI.Icons" Version="3.0.0-RC.1" />
```

but okay with 
```csproj
<PackageReference Include="Microsoft.Fast.Components.FluentUI" Version="3.0.0-RC.1" />
<PackageReference Include="Microsoft.Fast.Components.FluentUI.Icons" Version="3.0.0-preview.5" />
```

**Comment by vnbaaij** [comment]
Ok, then just use the preview-5 Icons. There were no changes between this preview and RC1

**Comment by dvoituron** [comment]
Indeed, that's an error in the referenced assemblies in [FluentUI.Icons 3.0.0-RC.1 lib](https://nuget.info/packages/Microsoft.Fast.Components.FluentUI.Icons/3.0.0-RC.1).

```
Full Name: Microsoft.Fast.Components.FluentUI.Icons
AssemblyFileVersion:3.0.23198.3
Referenced assemblies: Microsoft.Fast.Components.FluentUI, Version=3.1.23185.1
```

We will fix that in the next version.

In the meantime, as Yue and Vincent suggested, you can reference version `3.0.0-preview.5` which is exactly the same content.

**Comment by StevenRasmussen** [comment]
BTW - When using the latest VS2022 preview and .Net 8 this was causing VS to hang and then eventually crash :(

**Comment by vnbaaij** [comment]
This is fixed with final 3.0 packages
